Modules/vc-module-pricing/VirtoCommerce.PricingModule.Data/ExportImport2/PricelistExportDataQuery.cs

```diff
@@ -6,6 +6,13 @@ namespace VirtoCommerce.PricingModule.Data.ExportImport
 {
     public class PricelistExportDataQuery : ExportDataQuery
     {
+        public PricelistExportDataQuery()
+        {
+            IncludedProperties = new[]
+            {
+                "Id", "Name", "Currency", "Assignments.Id", "Assignments.CatalogId", "Assignments.PricelistId", "Assignments.Name", "Assignments.Priority", "Assignments.ConditionExpression","Assignments.PredicateVisualTreeSerialized", "Prices.Id", "Prices.PricelistId", "Prices.Currency", "Prices.ProductId", "Prices.Sale", "Prices.List", "Prices.MinQuantity","Prices.StartDate", "Prices.EndDate", "Prices.EffectiveValue" };
```

Comment:
Let's exclude these:
"Assignments.ConditionExpression","Assignments.PredicateVisualTreeSerialized"
And probably no need in "Assignments.PricelistId" and "Prices.PricelistId"